src/databricks/labs/remorph/reconcile/connectors/source_adapter.py

```diff
@@ -0,0 +1,21 @@
+from pyspark.sql import SparkSession
+from sqlglot import Dialects
+
+from databricks.labs.remorph.config import SQLGLOT_DIALECTS
+from databricks.labs.remorph.reconcile.connectors.data_source import DataSource
+from databricks.labs.remorph.reconcile.connectors.databricks import DatabricksDataSource
+from databricks.labs.remorph.reconcile.connectors.oracle import OracleDataSource
+from databricks.labs.remorph.reconcile.connectors.snowflake import SnowflakeDataSource
+from databricks.sdk import WorkspaceClient
+
+
+class DataSourceAdapter:
```

Comment:
We can create a simpler `__main__.py` to load the necessary engine. @vijaypavann-db has handled this in other PR check with him

Reply:
sure will check that.

Reply:
We could simply keep the function in this file why to keep it in a class